openapi/hubspot.analytics/client.bal

```diff
@@ -0,0 +1,77 @@
+import  ballerina/http;
```

Comment:
We should add licence header in the top of any .bal file or .java file

Reply:
Fixed. Thanks for pointing out